
Failed to establish dbus connection
global_step=199, episodic_return=-1783.799560546875
global_step=199, episodic_cost=0
global_step=399, episodic_return=-1633.66943359375
global_step=399, episodic_cost=0
global_step=599, episodic_return=-1489.798828125
global_step=599, episodic_cost=3
global_step=799, episodic_return=-1573.595703125
global_step=799, episodic_cost=0
global_step=999, episodic_return=-1545.6153564453125
global_step=999, episodic_cost=0
global_step=1199, episodic_return=-1567.5865478515625
global_step=1199, episodic_cost=1
global_step=1399, episodic_return=-1612.8719482421875
global_step=1399, episodic_cost=0
global_step=1599, episodic_return=-1727.0263671875
global_step=1599, episodic_cost=0
global_step=1799, episodic_return=-1749.4632568359375
global_step=1799, episodic_cost=0
global_step=1999, episodic_return=-1342.09521484375
global_step=1999, episodic_cost=7
global_step=2199, episodic_return=-1680.410888671875
global_step=2199, episodic_cost=0
global_step=2399, episodic_return=-1476.8111572265625
global_step=2399, episodic_cost=7
global_step=2599, episodic_return=-1498.45703125
global_step=2599, episodic_cost=2
global_step=2799, episodic_return=-1583.6783447265625
global_step=2799, episodic_cost=0
global_step=2999, episodic_return=-1526.350341796875
global_step=2999, episodic_cost=0
global_step=3199, episodic_return=-1549.3187255859375
global_step=3199, episodic_cost=0
global_step=3399, episodic_return=-1502.3277587890625
global_step=3399, episodic_cost=6
global_step=3599, episodic_return=-1324.3731689453125
global_step=3599, episodic_cost=9
global_step=3799, episodic_return=-1594.95947265625
global_step=3799, episodic_cost=0
global_step=3999, episodic_return=-1276.828369140625
global_step=3999, episodic_cost=13
global_step=4199, episodic_return=-1656.2222900390625
global_step=4199, episodic_cost=0
global_step=4399, episodic_return=-1551.653564453125
global_step=4399, episodic_cost=0
global_step=4599, episodic_return=-1611.3045654296875
global_step=4599, episodic_cost=0
global_step=4799, episodic_return=-1368.7342529296875
global_step=4799, episodic_cost=12
global_step=4999, episodic_return=-1393.270751953125
Failed to establish dbus connectionTraceback (most recent call last):
  File "sac.py", line 497, in <module>
    writer.add_scalar("losses/qfc_loss", qfc_loss.item(), global_step)
NameError: name 'qfc_loss' is not defined
[ 1.3003006  -4.04290208]
[ 0.96187589 -2.76705316]
[0.82764788 2.42715004]
[1.12096804 3.47642266]
[1.50824444 4.29064458]